assembl/fabfile.py

```diff
@@ -2688,3 +2688,16 @@ def set_fail2ban_configurations():
         finally:
             for path in filters_to_file.values():
                 os.unlink(path)
+
+
+@task
+def install_jq():
```

Comment:
Put this on the main system installation trail. Likely in `install_server_deps`, this should ensure it's installed on future installs. It can also be a single command run to ensure that servers are migrated properly.